### ISSUE TITLE
Notification.new is not a function

### ISSUE THREAD

**Issue by oleklapp** [EddyVerbruggen/nativescript-local-notifications#18]
I have updated to nativescript to 2.3 and the newest xcode. I then updated this plugin, but now Im getting the following error when trying to use it:

file:///app/tns_modules/nativescript-local-notifications/local-notifications.js:26:41: JS ERROR TypeError: Notification.new is not a function. (In 'Notification.new()', 'Notification.new' is undefined)

**Comment by EddyVerbruggen** [comment]
😢  fixed in 1.1.4, thanks for letting me know!

**Comment by oleklapp** [comment]
This error might be because of something else entirely actually. I dont know if its just me or not. 
Its actually the variable Notification that is empty in my case.
console.dump(Notification); returns the following:

=== dump(): dumping members ===
	0
	=== dump(): dumping function and properties names ===
	=== dump(): finished ===

thus I am also getting:
JS ERROR TypeError: undefined is not an object (evaluating 'Notification.alloc().init')

**Comment by EddyVerbruggen** [comment]
To be honest I'm a bit puzzled why those lines are there at all.

Can you comment [these lines](https://github.com/EddyVerbruggen/nativescript-local-notifications/blob/master/local-notifications.ios.js#L26-L27) and try the plugin?

**Comment by oleklapp** [comment]
I tried reinstalling the app and got it to work. And sorry, the Notification.new() and NotificationManager.new() was the correct alternative in the newer iOS. 

However, Im still having some issues with the last release:
CONSOLE LOG file:///app/tns_modules/nativescript-local-notifications/local-notifications.js:72:18: Error in LocalNotifications.requestPermission: TypeError: null is not an object (evaluating 'app.currentUserNotificationSettings')

This is error comes from line 89. I think after having a look at UIApplication yesterday that sharedApplication now is deprecated. When I googled it I saw that UIApplication.sharedApplication now has been renamed to UIApplication.shared

Also, the UIApplication.shared doesnt contain the method currentUserNotificationSettings anymore.
